### ISSUE TITLE
Inference time

### ISSUE THREAD

**Issue by YJonmo** [zju3dv/LoFTR#263]
Has anyone tried to get this running using tensorRT or C++?

It would be good to know the rough fps for this.

Cheers,

**Comment by oooooha** [comment]
> 
Have you tried it now? Because I'm also thinking about it

**Comment by YJonmo** [comment]
No I gave up. Instead I am using Light Glue. LoFTR does not provide descriptors which I need for my work.

**Comment by oooooha** [comment]
> u

I have now converted the weights of Outdoor and indoor to the ONNX model and tested the time, and the results have not changed much from the Torch model, and the TRT model has not been successful

**Comment by YJonmo** [comment]
Maybe you could try this:
https://github.com/Kolkir/Coarse_LoFTR_TRT

But I prefer this one: https://github.com/cvg/LightGlue

**Comment by nuaayxy** [comment]
i think with lightglue you would have some issue with low texture, its not that robust to low texture setting

**Comment by YJonmo** [comment]
true, now I am using the dual-softmax+DISK (epipolar), not as good as LoFTR for low texture area but better than Superpoint and LighGlue.

**Comment by zoey9628** [comment]
> > u
> 
> I have now converted the weights of Outdoor and indoor to the ONNX model and tested the time, and the results have not changed much from the Torch model, and the TRT model has not been successful

How did you convert the model to the onnx model? Would you mind sharing your code? I would really appreciate it.
